.editorconfig

```diff
@@ -1,8 +0,0 @@
-# EditorConfig is awesome: http://EditorConfig.org
-root = true
-
-[*]
-end_of_line = lf
-insert_final_newline = true
-indent_style = space
-indent_size = 2
```

Comment:
this can be helpful for convincing VS Code to pick the right auto-indent level when opening files ... I know prettier will ultimately take care of it but it's still nice to have the indent set right.

Reply:
Yeah actually we had a bit of discussion about this last week. Does vscode respect editorconfig out of the box or does it require a plugin?

if we only care about vscode, maybe it’s easier to change .vscode/settings.json?

Either way we should figure out what we want to do and add it to the upstream template repo.

Reply:
I think it requires this extension, which is very popular; we could add it to our recommended extensions: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

Reply:
How about this?

https://github.com/foxglove/template-typescript/pull/34

Seems simpler than asking people to install yet another plugin. Obviously not cross-editor compatible, but at this point we aren't officially supporting multiple editors anyway.